include/glow/Importer/CommonOperatorLoader.h

```diff
@@ -218,8 +218,31 @@ class CommonOperatorLoader : public ProtobufLoader {
     } else {
       Tensor *T = getTensorByName(op.input(1));
       auto TH = T->getHandle<size_t>();
+
```

Comment:
Can we add an assertion here that the shape of T is `{in.dims()}`?

Reply:
I don't think we can assume that the number of elements in T will be the same as the number of dimensions in 'in'. The description in ONNX and Caffe2 don't request this. For instance, we may want to reshape [1, 1, 1, 200] into [200], like does the Squeeze operator.

Reply:
Oops yeah you're right, never mind 😄